chart/templates/persi-broker.yaml

```diff
@@ -43,9 +43,11 @@ spec:
                 - plan_id: {{ .id }}
                   plan_name: {{ .name }}
                   description: {{ .description }}
-                  kube_storage_class: {{ .kube_storage_class }}
+		  {{ with .kube_storage_class }}
```

Comment:
Nit: tab in file (GitHub diff is showing it as red).

Not a blocker: would be nice to just merge #9 instead; this doesn't really need to exist.

Reply:
Actually I believe that it would be a blocker.
This is YAML in a string, and is treated as YAML later on, at which point things break.
I thought I had killed all the TABs.
Will fix.
I haven't tested that #9 works.

Done with commit 787e75f30ec4784f5450430939dcff98c8a30ad6
Had it fixed, fix was lost in git staging before commit :(